### PR TITLE
cloud_defend index read priv added to kibana_system role (for usageCollection)

### PR DESCRIPTION
**Pull request by mitodrummer**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)? y
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md)? y
- If submitting code, have you built your formula locally prior to submission with `gradle check`? y
- If submitting code, is your pull request against main? Unless there is a good reason otherwise, we prefer pull requests against main and will backport as needed. y
- If submitting code, have you checked that your submission is for an [OS and architecture that we support](https://www.elastic.co/support/matrix#show_os)? y
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

File: x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStoreTests.java
Comment (slobodanadamovic) [inline x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStoreTests.java:1242]: Can we also assert that `view_index_metadata` is granted as well?
```suggestion
            assertViewIndexMetadata(kibanaRole, indexName);
        });
```

**Review by slobodanadamovic — APPROVED** [review]
Looks good from `es-security` perspective, but you still need approval from 
@elastic/kibana-security team because this change affects `kibana_system` role.

**Comment by jeramysoucy** [comment]
@mitodrummer Could you tell us a bit more about this index, who owns it and what it is used for?

**Comment by mitodrummer** [comment]
oh sorry, i didn't see your comment and merge a little pre maturely.

**Comment by mitodrummer** [comment]
Indexes are owned by this integration: https://github.com/elastic/integrations/tree/main/packages/cloud_defend which is under the cloud security team

File: x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStoreTests.java
Comment (mitodrummer) [reply x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStoreTests.java:1242]: Sorry I saw the approval and merged. Didn't notice this comment. I can open another PR to make this change if you like.

**Comment by jeramysoucy** [comment]
@mitodrummer Thanks. What is the `kibana_system` use case for the two indices? You mentioned usageCollection, but there is not a lot of information in the PR description to provide context.

File: x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStoreTests.java
Comment (slobodanadamovic) [reply x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStoreTests.java:1242]: Yes, it would be nice if you could add it.

**Review by jeramysoucy — COMMENTED** [review]
Just leaving a retroactive review to note that Kibana Security is approving of the changes to the `kibana_system` role in this case as the purpose is specifically to aid telemetry. Thanks!

**Pull request merged**
